Arm/rover_arm_system.hpp

```diff
@@ -3,29 +3,43 @@
 #include "joint.hpp"
 #include "Hand/wrist_joint.hpp"
 #include "../Common/rover_system.hpp"
+#include <cmath>
 
 namespace sjsu::arm
 {
 class RoverArmSystem : public sjsu::common::RoverSystem
 {
  private:
-  // isOperational defines if the arm should be allowed to move or if it should
-  // be in its resting 'off' position.
-  bool isOperational;
-
   sjsu::arm::Joint & Rotunda;
   sjsu::arm::Joint & Shoulder;
   sjsu::arm::Joint & Elbow;
   sjsu::arm::WristJoint & Wrist;
```

Comment:
```suggestion
  sjsu::arm::Joint & rotunda_;
  sjsu::arm::Joint & shoulder_;
  sjsu::arm::Joint & elbow_;
  sjsu::arm::WristJoint & wrist_;
```
Also move these private members to the bottom of class since this is the standard for C++

Reply:
member variables are `usually_ended_with_a_underscore_` like so